src/smartcontracts/code.ts

```diff
@@ -25,4 +37,12 @@ export class Code {
     valueOf(): Buffer {
         return Buffer.from(this.hex, "hex");
     }
+
+    computeHash(): Hash {
```

Comment:
It's probable that we'll have some sort of a `computer` object at some point (see the specs for transaction, and hash computation). But for now, it's perfect :+1: